Makefile

```
@@ -45,12 +45,12 @@ MAKEFLAGS += --no-print-directory
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
 HOSTGO := env -u GOOS -u GOARCH -u GOARM -- go
-
-LDFLAGS := $(LDFLAGS) -X main.commit=$(commit) -X main.branch=$(branch) -X main.goos=$(GOOS) -X main.goarch=$(GOARCH)
+BUILD_INFO_IMPORT_PATH=github.com/influxdata/telegraf/internal
+LDFLAGS := $(LDFLAGS) -X $(BUILD_INFO_IMPORT_PATH).commit=$(commit) -X $(BUILD_INFO_IMPORT_PATH).branch=$(branch) -X $(BUILD_INFO_IMPORT_PATH).goos=$(GOOS) -X $(BUILD_INFO_IMPORT_PATH).goarch=$(GOARCH)
```

We no longer have `internal.goos` and `internal.goarch` so I think this should be (with shortened variable):
```suggestion
INTERNAL_PKG=github.com/influxdata/telegraf/internal
LDFLAGS := $(LDFLAGS) -X $(INTERNAL_PKG).commit=$(commit) -X $(INTERNAL_PKG).branch=$(branch)
```
If you accept the shortened name, please also change below!

---

@srebhan this is where you come by and say yes :)

---

Ummm yes? :-) I'm not against leaving it in, but I'm not sure what happens if there is no such variable defined? What is the purpose of this?

---

The purpose of leaving it in is to keep the current behavior, but more importantly, to clearly see that the variable is set or when it is set when running build commands.

---

@srebhan and I chatted and I think we now agree on removing GOOS and GOARCH for now.

I think we only need to replace all the `BUILD_INFO_IMPORT_PATH` with `INTERNAL_PKG` as @srebhan suggested above.